go.mod

```diff
@@ -30,6 +30,7 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.11.0
 	github.com/shirou/gopsutil v3.21.8+incompatible
+	github.com/soothing-rain/gorocksdb v0.0.0-20220113075731-e68e68ed4c62 // indirect
```

Comment:
Are we using a forked gorocksdb to make the compilation on mac work, or is this a mistake?

Reply:
We use RocksDB v6.27.3 (instead of v6.15.2) when building with Mac, as the old version does not support the Apple Silicon chip. In the meanwhile, we are using a forked (and modified) version of gorocksdb client when building with Mac.

Reply:
Recommended to  fork it with `milvus-io` Github Account

Reply:
This is a temporary solution,  the next step is to upgrade gorocksdb, any ideas? @soothing-rain

Reply:
Yeah I think it makes sense to migrate to milvus-io before merge.

Reply:
May we push the gorocksdb repo to upgrade?

Reply:
Perhaps we can file an issue in their repo? If that project is still actively maintained...